valueformatting.go

```diff
@@ -181,46 +181,46 @@ func (formatting *valueFormatting) pbFormatter(ctype string) (valueFormatter, er
 	}, nil
 }
 
-type validEncodings string
+type ValidEncodings int
```

Comment:
Is there a reason to make this exported?

Reply:
No, not really. I was trying to appease the tests. Changed.